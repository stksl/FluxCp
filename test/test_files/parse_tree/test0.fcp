
ProgramBound
	StructDefine
		StructField
		FunctionDeclaration
			FunctionHeader
				FunctionArgument
				FunctionArgument
			BodyBound
				EmptySyntaxNode