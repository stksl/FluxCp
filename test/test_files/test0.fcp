## test case 0 ##

struct   int 
     {
                    int instance;
    
    int somefunction(int      arg1    ,int arg2)  {
      }
}

