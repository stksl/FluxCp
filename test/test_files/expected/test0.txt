
ProgramBound
	StructDefine
		StructField
		FunctionDeclaration
			FunctionHeader
				FunctionArgument
				FunctionArgument
			FunctionBodyBound
				EmptySyntaxNode